### PR TITLE
Fix istio config for v1.9.1

### LINKED ISSUE #45: Wrong example
https://github.com/cert-manager/istio-csr/blob/9a223f9222924a5d65b90742391f293c900c7ce2/hack/istio-config-1.9.1.yaml#L52-L55

Duplicate _name_.

### PR DESCRIPTION
**Pull request by m1pl**
fixes #45

### CODE REVIEW

**Comment by m1pl** [comment]
CONTRIBUTING.md is missing...

File: hack/istio-config-1.9.1.yaml [hunk 1]
```diff
@@ -50,6 +50,6 @@ spec:
             value:
               name: ca-root-cert
               configMap:
-                name: istiod-tls
+                secretName: istiod-tls
```
Comment (JoshVanL) [inline hack/istio-config-1.9.1.yaml:53]: Thanks! I think this line needs to be removed completely?

**Review by JoshVanL — CHANGES_REQUESTED** [review]
Thanks for this!

I will add a `CONTRIBUTING.md`.

/approve
/ok-to-test

File: hack/istio-config-1.9.1.yaml (hunk 1 above)
Comment (m1pl) [reply hack/istio-config-1.9.1.yaml:53]: @JoshVanL I copied it from the old version: https://github.com/cert-manager/istio-csr/blob/dacb33e62cd33ff6f5787d371de35666d5d29511/hack/istio-config-1.8.2.yaml#L53

Comment (JoshVanL) [reply hack/istio-config-1.9.1.yaml:53]: Yep, I think it is also wrong there as well.

Comment (m1pl) [reply hack/istio-config-1.9.1.yaml:53]: So you want me to remove all three lines, or just the one for 1.9.1?

**Comment by m1pl** [comment]
@JoshVanL I removed the line from all 3 files.

**Comment by JoshVanL** [comment]
Thanks @m1pl!

/lgtm

**Pull request merged**